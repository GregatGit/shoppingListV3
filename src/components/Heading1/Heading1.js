import React from 'react'
import './Heading1.css'

const Heading1 = (props) => (
  <div className='myCenter'>
    <h1>
      Welcome to your Shopping List
    </h1>
    <h2>
      Select shopping list(s)
    </h2>
  </div>
)

export default Heading1
