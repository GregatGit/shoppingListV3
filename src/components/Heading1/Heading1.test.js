import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Heading1 from './Heading1'

test('Header component should render', () => {
  const component = shallow(<Heading1 />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
