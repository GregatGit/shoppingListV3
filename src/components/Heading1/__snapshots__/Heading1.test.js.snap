// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render 1`] = `
<div
  className="myCenter"
>
  <h1>
    Welcome to your Shopping List
  </h1>
  <h2>
    Select shopping list(s)
  </h2>
</div>
`;
